tools/run-mypy

```diff
@@ -39,6 +39,15 @@ zerver/tests/test_upload.py
 zerver/tests/test_narrow.py
 """.split()
 
+# We don't run mypy on contrib_bots, since the code there will
+# often be shared with other projects that do not want a mypy
+# dependency (at least while it's still kind of beta).
+exclude_dirs = """
+contrib_bots
+""".split()
+
+exclude_common += exclude_dirs
```

Comment:
I think `exclude_common` accepts directories already, so I think you don't need to do this as a separate variable